_posts/2025-04-28-visualizar-simulaciones-advance.md

```
@@ -0,0 +1,56 @@
+
+<p align="center">
+  <img src="https://cdn.capitalexpress.cl/img/panel_principal_simulaciones.jpg" alt="menu_simulaciones">
+</p>
+
+Pero, ¿Qué cosas puedes hacer en esta ventana?, a continuación te mostraremos a detalle como cambiar plazo de facturas, descargar carta guía, cambiar el estado de una solicitud.
+
+### Cambiar estado de una simulación
+
+Cuando tienes una simulación contará con un estado predeterminado que es `nueva` simulación, el cual debe ser cambiado manualmente dependiendo de la condición a la que se quiera pasar, teniendo a disposición los estados `Enviada` y `Abandonada`, según sea el caso, ¿Cómo hacerlo?, aquí te dejamos un video explicando como se hace.
+
+<video width="100%" controls>
```

markdownlint-cli2 (0.17.2)

Inline HTML
Element: video

(MD033, no-inline-html)